### ISSUE TITLE
Installing pear/net_smtp with composer & --prefer-stable fails

### ISSUE THREAD

**Issue by demon** [pear/Net_SMTP#24]
Due to a lack of stable releases for pear/net_socket. 

```
  Problem 2
    - Installation request for pear/net_smtp 1.7.1 -> satisfiable by pear/net_smtp[1.7.1].
    - pear/net_smtp 1.7.1 requires pear/net_socket * -> no matching package found.
```

I know this is technically an issue against pear/net_socket, but issues are disabled there and the project seems mostly unmaintained.

**Comment by alecpl** [comment]
@jparise This probably is not a bug anymore, as I see Net_Socket has stable releases. However, how about:
```
"pear/pear-core-minimal": "~1.10",
"pear/net_socket": "~1.2"
```
or `@stable`?

**Comment by jparise** [comment]
Yes, it would be nice to close this and move to a newer stable release. Can you send a pull request for this?

**Comment by alecpl** [comment]
@jparise This can be closed. How about a new release?

**Comment by jparise** [comment]
@alecpl I just released [version 1.8.1](https://github.com/pear/Net_SMTP/releases/tag/1.8.1).

**Comment by alecpl** [comment]
@jparise Thank you very much.
